### PR TITLE
G-like popup

### PR DESCRIPTION
**Pull request by CrazyPike**
### HTML CSS Popup

[Demo](https://crazypike.github.io/frontend-2019-homeworks/submissions/CrazyPike/htmlcsspopup/index.html) |
[Code base](https://github.com/CrazyPike/frontend-2019-homeworks/tree/master/submissions/CrazyPike/htmlcsspopup)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/CrazyPike/htmlcsspopup/index.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
```
Comment (A-Ostrovnyy) [inline submissions/CrazyPike/htmlcsspopup/index.html]: Don't forget about `meta` for mobile devices.

File: submissions/CrazyPike/htmlcsspopup/index.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    <div class="menu__item">Images</div>
+    <div class="menu__item">
+    <form id="menuform" class="menu__popup-service-from">
+    <input type="checkbox" form="menuform" class='menu__open-popup-visible' id="menu__popup__visible"/>
+        <label  class="menu__popup-icon"  for="menu__popup__visible"></label>
+    <button form="menuform" type="reset" class="menu__close-popup-button" tabindex="1"></button>
+    <div class="menu__arrow-to-popup"></div>
+    <div class="popup">
+
+        <div class="popup__item" tabindex="0"><img class="popup__img" src="img/docs.png"><span class='popup__label'>Docs</span></div>
```
Comment (A-Ostrovnyy) [inline submissions/CrazyPike/htmlcsspopup/index.html]: Fill `alt` attribute of `` tag.

File: submissions/CrazyPike/htmlcsspopup/styles.css [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+    transform: rotate(45deg);
+    position: relative;
+    left: 0px;
+    top: -10px;
+    background-color: white;
+    display: none;
+
+}
+
+
```
Comment (A-Ostrovnyy) [inline submissions/CrazyPike/htmlcsspopup/styles.css]: Remove unused lines

**Comment by CrazyPike** [comment]
Thanks you for your answer! 
Fixes are done.

**Comment by A-Ostrovnyy** [comment]
Your menu icon without outline/border on focus state. Also, Menu icon jumped up when popup is opened [screen](https://imgur.com/a/w0kd4F5), and on Windows Chrome  your popup looks like [that](https://imgur.com/a/HxFymTI).
Please fix these issues.

**Comment by CrazyPike** [comment]
Fixed, the width of scroll is different in windows and mac. And added fix for focus

**Comment by A-Ostrovnyy** [comment]
Nice work!

**Pull request merged**